internal/controller/components/kserve/kserve_controller_actions_test.go

```diff
@@ -479,3 +485,30 @@ func TestCleanUpTemplatedResources_withoutAuthorino(t *testing.T) {
 		),
 	)
 }
+
+func TestCheckPreConditions_Managed_NoLWSOperator(t *testing.T) {
+	ctx := t.Context()
+	g := NewWithT(t)
+
+	cli, err := fakeclient.New()
+	g.Expect(err).ShouldNot(HaveOccurred())
+
+	ks := componentApi.Kserve{}
+	ks.Spec.Serving.ManagementState = operatorv1.Managed
+
+	dsci := dsciv2.DSCInitialization{}
+	dsci.Spec.ServiceMesh = &infrav1.ServiceMeshSpec{
+		ManagementState: operatorv1.Managed,
+	}
```

Comment:
i dont think we need this check as servicemesh will be taken away from dsci soon